automatic_push/iana_push.py

```diff
@@ -66,17 +69,15 @@ def __init__(self, config: ConfigParser = create_config()):
         log_directory = self.config.get('Directory-Section', 'logs')
         self.logger = log.get_logger('iana_push', f'{log_directory}/jobs/iana-push.log')
 
-        self.repo: t.Optional[repoutil.ModifiableRepoUtil] = None
-
     @job_log(file_basename=BASENAME)
     def __call__(self) -> list[JobLogMessage]:
         self.logger.info('Starting job to push IANA-maintained modules')
-        self.repo = get_forked_repository(self.config, self.logger)
+        self.repo = get_forked_worktree(self.config, self.logger)
```

Comment:
shouldn't the repo be removed after executing the script as in the ```ietf_push.py``` repoutil.remove_worktree(self.repo.working_dir)?

Reply:
it is, on line 96